exasol/slc/internal/tasks/test/test_runner_db_test_base_task.py

```diff
@@ -236,42 +238,38 @@ def get_result_status(status):
                     break
         return result_status
 
-    def get_test_folders(self, test_config):
-        test_folders = []
-        if test_config["test_folders"] != "":
-            test_folders = test_config["test_folders"].split(" ")
-        if self.tests_specified_in_parameters():
-            test_folders = self.test_folders
```

Comment:
do not remove this:
With this check you can still override the ci.json test specification with CLI arguments. 
This is very useful when you need to run a specific test locally.